### ISSUE TITLE
Banners don't wave like in vanilla

### ISSUE THREAD

**Issue by ZeLalo** [sp614x/optifine#2555]
As the title says. Banners don't appear to be waving like they do in the vanilla version. I have tried messing with a bunch of settings but nothing seems to work.

**Comment by sp614x** [comment]
Banners are waving normally for me.
Which version, runing standalone or with forge/fabric?

**Comment by ZeLalo** [comment]
Okay so. I did some more investigation. And I found out that they wave perfectly fine in survival. But the issue is only in multiplayer afaik. I am running the latest release as of now (1.14.3 HD U F1) and standalone. The video clip is if F3 data is needed. And the resource pack I'm using is a custom one that only changes a few textures and should not interfere with banners in any way. I did also test without the resource pack, same thing, they're not waving.
[BannerNotWAvingInMP.zip](https://github.com/sp614x/optifine/files/3342222/BannerNotWAvingInMP.zip)
Here are my settings too if they are needed:
[options.txt](https://github.com/sp614x/optifine/files/3342224/options.txt)
[optionsof.txt](https://github.com/sp614x/optifine/files/3342225/optionsof.txt)
